stream-chat-swift-docs/docs/guides/working-with-a-channel.md

```diff
@@ -1,3 +1,298 @@
 ---
 title: Working with a Channel
----
\ No newline at end of file
+---
+
+## Getting Channel Data
+
+Here's how you can get data of a Channel you already know the Channel ID of:
+
+```swift
+let channelController = chatClient.channelController(for: ChannelId(type: .messaging, id: <#Your Channel ID#>)
```

Comment:
What we discussed the other day on snippets width applies to almost all the snippets here, they don't fit my MBP16" screen 😢
<img width="345" alt="Screenshot 2021-05-21 at 18 32 38" src="https://user-images.githubusercontent.com/7489425/119162716-5ecdb400-ba63-11eb-863d-0d6668d35e4e.png">